models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -501,10 +477,9 @@ aeif_cond_beta_multisynapse::calibrate()
   assert( V_.RefractoryCounts_
     >= 0 ); // since t_ref_ >= 0, this can only fail in error
 
-  B_.spikes_.resize( P_.num_of_receptors );
+  B_.spikes_.resize( P_.n_receptors() );
   S_.y_.resize( State_::NUMBER_OF_FIXED_STATES_ELEMENTS
-    + ( State_::NUMBER_OF_STATES_ELEMENTS_PER_RECEPTOR
-                  * P_.num_of_receptors ) );
+    + ( State_::NUMBER_OF_STATES_ELEMENTS_PER_RECEPTOR * P_.n_receptors() ) );
```

Comment:
I think it would be good to explicitly give the initial value `0` as second argument to `resize()`.